index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ro">
+    <meta charset="UTF-8">
+    
+    <head class="header">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Save the planet</title>
+        <link href="./CSS/homestyle.css" rel=stylesheet>         
+    </head>
+    <body>
+        <section class="coverContainer">
+            <div class="coverImage">
+                <div class="shape"><img  src="./Photos/Shape.png" alt="shape"></div>
+                <div class="coverContent">
+                    <div class="subtitle">
+                        <div class="ractangleImg"><img  src="./Photos/Rectangle.png" alt="rectangle"></div>
+                        WE TEACH YOU TO PROTECT OUR   
+                    </div>
+                    <div class="title">PLANET</div>
+                    <div class="description">
+                        Our planet is precious and vital for our<br>
+                        beautiful lives, if we don’t take care of it<br>
+                        it will dissapear along with us.
+                    </div>  
+                    <div class="mention">
+                        *Please take note that your actions matter more than you think.
+                    </div>
+                    <div class="coverActions"> 
+                        <button class="buton1">Find out more</button>              
```

Comment:
aici ar trebui sa folosesti un tag de a
Iti faci o clasa generala de button care contine cam tot ce ar trebui sa contina clasa unui buton, ex. cursor: pointer, o translatie, chestii de genul. Apoi ii pui o variatie de --green in care schimbi ce e specific butonului tau, ca e verde si are padding diferit, ca are hover diferit, etc. Iar pe clasa de hero__button te ocupi de pozitionarea sa in interiorul hero-ului
```suggestion
                        <a href="#" class="hero__button button--green">
                                      Find out more
                        </a>
```